app/mixins/message-wall-reply-notification.js

```diff
@@ -0,0 +1,48 @@
+import Mixin from '@ember/object/mixin';
+
+export default Mixin.create({
+  /**
+  * Constructs a localized reply notification body
+  * @param {Ember.Object} model
+  * @returns {string}
+  */
+  getMessageWallReplyBody(model) {
+    const totalUniqueActors = model.get('totalUniqueActors');
+    const hasTwoOrMoreUsers = totalUniqueActors > 1;
+    const firstReplierName = model.get('latestActors.0.name');
+
+    // need this check 
+    const isOwnWall = firstReplierName === '// check model.uri for name in Message Wall uri';
```

Comment:
The wall owner isn't `latestActors.0`, that's the first person replying. Same goes for the original wall post (file above). We need to get the current logged in user's name, wherever that's stored in mobile-wiki, and compare it to the username in the wall URI.

Reply:
Correct. It looks like we can use `this.usernameMarkup` to retrieve **username**.